### DISCUSSION TITLE
Big fan over here!

### DISCUSSION THREAD

**Discussion by jazFromHouston** [berarma/ffbtools#36] [Q&A] [answered]
I signed up for GitHub just to reach out to you. The OverSteer program works well with my Logitech G923 (XBox/PC version) but I really dislike gear driven clickity clacking wheels.

The only thing keeping me from Linux 100% is my Moza R5 base. The direct drive wheel is incredible, and the FFB is unfortunatly non existent in Fedora Linux. I can get my pedals, and the wheel along with the buttons to work OOTB, but the FFB is a no go.

I wanted to work with you to provide logs and whatever else I could to get more people over from dirty windoze to Linux that love Sim racing.

Cheers!

-Jay

**Comment by leillo1975** [comment]
Hi, in addition to what @berarma can tell you, I recommend you ask in this Matrix chat room:
https://matrix.to/#/#simracer:matrix.org

I know that there are people out there who are in your situation and are working to support your base. I don't remember the name of the user who is working on it, but you can ask and give your feedback there too.

**Comment by berarma** [comment]
Thank you for your kind words.

I've found [this report](https://github.com/libsdl-org/SDL/issues/8422) about the Moza R5 working on Linux, FFB included.

I guess it will use the HIDPP driver on Linux which should give support to most modern mid/high-end wheels. You can check which driver is being loaded, if any, checking the output of the `dmesg` command.

There's one issue with that driver though which might be the cause for the missing FFB. It doesn't correctly play effects with length zero, and most games use it this way. For most other drivers and APIs, length zero means playing indefinitely, but not for this one.

The [`ffbwrap`](https://github.com/berarma/ffbtools/blob/master/docs/ffbwrap.md) command in this package implements a workaround in the `--duration-fix` option. If you could try it we could know if that's the case. In the linked page there are instructions to use the command.

**Comment by jazFromHouston** [comment]
Hey you all. Thank you! 

I hadnt booted into Linux for around 8-9 days, and lo-and-behold 8 days ago someone released a project for the R3 all the way to the R-21. With some struggle, I got FFB and my wheel buttons working on my MOZA R5 base,and the RS V2 wheel. I followed their instructions to a T, but for whatever reason, PitHouse will not detect anything.

It no longer crashes anymore upon launching, but I am using WINECFG through Lutris to launch it. Other than being able to configure the wheel for all my games with Pit House, I just have to do it for each game. Once it's set, its good.

No need for Windows anymore.

**Reply by poggersb** [reply]
What project is this, do you mind sharing?

**Reply by berarma** [reply, marked as answer]
> What project is this, do you mind sharing?

https://github.com/JacKeTUs/moza-ff
